tests/end2end-tests/graphstorm-nc/test.sh

```diff
@@ -101,6 +101,13 @@ if [ ! -f ./models/movielen_100k/train_val/movielen_100k_ngnn_model/GRAPHSTORM_R
     exit 1
 fi
 
+# Ensure a file named data_transform_new.json was copied under --save-model-path
+if [ ! -f ./models/movielen_100k/train_val/movielen_100k_ngnn_model/data_transform_new.json ]; then
+    echo "data_transform_new.json was not copied from input data"
+    exit 1
+fi
+
```

Comment:
Same as above.

Reply:
This tests whether the train job copied over the GConstruct file, we're not testing a GConstruct job.